pkg/agent/controller/noderoute/node_route_controller_test.go

```diff
@@ -631,3 +615,96 @@ func TestGetPodCIDRsOnNode(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoveStaleGatewayRoutes(t *testing.T) {
+	fakeClientSet := fake.NewSimpleClientset(node1)
+	c := newController(t, &config.NetworkConfig{}, fakeClientSet)
+	defer c.queue.ShutDown()
+
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+	c.informerFactory.Start(stopCh)
+	c.informerFactory.WaitForCacheSync(stopCh)
+
+	c.routeClient.EXPECT().Reconcile([]string{podCIDR.String()})
+	err := c.removeStaleGatewayRoutes()
+	assert.NoError(t, err)
+}
+
+func TestRemoveStaleWireGuardPeers(t *testing.T) {
+	node1 := &corev1.Node{
```

Comment:
`node1` is also used as global variable, so maybe use a different variable name here for better readability.

Reply:
Done